src/plugins/controls/timer/component/countdown.js

```diff
@@ -273,9 +279,15 @@ export default function countdownFactory($container, config) {
         })
         .on('render', function() {
             $time = $('.time', this.getElement());
+            $timeAriaLabel = $('#timerbox-aria-label', this.getElement());
+            const timeString = timeEncoder.encode(this.remainingTime / precision);
+            const timeArr = this.encodedTime.split(':');
```

Comment:
This is generating a runtime error: 
```javascript
TypeError: this.encodedTime is undefined' from http://act.tao/taoQtiTest/views/node_modules/@oat-sa/tao-test-runner-qti/dist/plugins/controls/timer/component/countdown.js?buster=5e73391bc0bd3:301:23 Error: "Caught[via window.onerror]: 'TypeError: this.encodedTime is undefined' from http://act.tao/taoQtiTest/views/node_modules/@oat-sa/tao-test-runner-qti/dist/plugins/controls/timer/component/countdown.js?buster=5e73391bc0bd3:301:23
```

Reply:
updated, added checking for missed properties